pkg/apis/networking/v1alpha1/service_types.go

```diff
@@ -53,16 +53,33 @@ type MultiClusterServiceSpec struct {
 	// +optional
 	Ports []ExposurePort `json:"ports,omitempty"`
 
-	// Range specifies the ranges where the referencing service should
-	// be exposed.
-	// Only valid and optional in case of Types contains CrossCluster.
-	// If not set and Types contains CrossCluster, all clusters will
-	// be selected, that means the referencing service will be exposed
-	// across all registered clusters.
+	// DiscoveryStrategy specifies how to sync the EndpointSlice from one cluster to another cluster.
+	// Default to be RemoteAndLocal, which means syncing the EndpointSlice from one cluster
+	// to another cluster from the beginning, the requests to this service will routed to all of them.
 	// +optional
-	Range ExposureRange `json:"range,omitempty"`
+	DiscoveryStrategy ServiceDiscoveryStrategy `json:"discoveryStrategy,omitempty"`
```

Comment:
1. In the proposal of MCS itself, there is no concept like `DiscoveryStrategy`.

2. This `Range` is used for the propagation range of serviceImport, it has nothing to do with `EndpointSlice`. So what's your reason for deleting this field?

Reply:
1. https://github.com/karmada-io/karmada/blob/master/docs/proposals/service-discovery/README.md  There is annotation: `discovery.karmada.io/strategy`

2. This field is not used, @XiShanYongYe-Chang  told me

Reply:
> Are you suggesting not introducing this field this time as it hasn't been designed yet, and we can take the default behavior( that is RemoteAndLocal) during implementation?

+1

Reply:
Why not remove it this time?

Reply:
cc @chaunceyjiang for conforming